### PR TITLE
Display error message on the console

### PR DESCRIPTION
**Pull request by neomatrix369**
Captured error messages and displayed in output and error consoles. Printed unsuccessful executions in output and error consoles.

### CODE REVIEW

File: src/main/java/com/tesco/couchbase/client/BetterCBQ.java [hunk 1]
```diff
@@ -60,6 +64,9 @@ public static void execute(String couchBaseHost, String bucketName, String query
       bucket = cluster.openBucket(bucketName);
       N1qlQueryResult queryResult = bucket.query(simple(queryString));
       errorOccurred = displayQueryResults(queryString, queryResult);
+    } catch (Exception ex) {
+      err.format(ERROR_MESSAGE_FORMAT, ex);
```
Comment (dhatanian) [inline src/main/java/com/tesco/couchbase/client/BetterCBQ.java]: Is this really convenient? You're printing the message twice. Also, does this print the stacktrace?

**Comment by neomatrix369** [comment]
@dhatanian have a look if you think its good.

File: src/main/java/com/tesco/couchbase/client/BetterCBQ.java [hunk 2]
```diff
@@ -81,6 +68,8 @@ public void execute(String couchBaseHost, String bucketName, String queryString)
       bucket = cluster.openBucket(bucketName);
       N1qlQueryResult queryResult = bucket.query(simple(queryString));
       errorOccurred = displayQueryResults(queryString, queryResult);
+    } catch (Exception exception) {
+      throw exception;
```
Comment (dhatanian) [inline src/main/java/com/tesco/couchbase/client/BetterCBQ.java]: Why ?

**Comment by dhatanian** [comment]
Nice! I think it's ready to merge!

File: src/main/java/com/tesco/couchbase/client/BetterCBQ.java (hunk 2 above)
Comment (neomatrix369) [inline src/main/java/com/tesco/couchbase/client/BetterCBQ.java]: Removed

**Pull request merged**